src/Symfony/Component/HttpClient/HttpClientTrait.php

```diff
@@ -471,10 +479,12 @@ private static function mergeQueryString(?string $queryString, array $queryArray
 
         if (null !== $queryString) {
             foreach (explode('&', $queryString) as $v) {
-                if ('' !== $v) {
-                    $k = urldecode(explode('=', $v, 2)[0]);
-                    $query[$k] = (isset($query[$k]) ? $query[$k].'&' : '').$v;
+                if ('' === $v) {
```

Comment:
Same here

Reply:
Same reason as in https://github.com/symfony/symfony/pull/31018#discussion_r273340920

Reply:
Im 👎

Reply:
Reverted.